packages/rainbowkit/src/wallets/walletConnectors/phantomWallet/phantomWallet.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface PhantomWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const phantomWallet = ({
```

Comment:
We're missing the `installed` property here which detects whether the extension is installed or not. Check the Metamask wallet connector to see an example of how this is done @nheingit

Reply:
not super familiar with pnpm monorepos. To do this in the same way that the other extensions do, we'd need to bump wagmi to `0.9.1` which got cut yesterday. This updates the `ethereum` object in wagmi to have the `isPhantom` property needed to stay in line with the nice typings.

I can override it with an `any`, but wanted to get your opinion there.

Also curious what does `installed` actually do? In the most recent deployment I don't find any unexpected behavior.